Marlin/ubl_motion.cpp

```diff
@@ -250,9 +231,8 @@
     const float m = dy / dx,
                 c = start[Y_AXIS] - m * start[X_AXIS];
 
-    const bool inf_normalized_flag = isinf(e_normalized_dist),
-               inf_m_flag = isinf(m);
-
+    const bool inf_normalized_flag=isinf(e_normalized_dist),
+               inf_m_flag=isinf(m);
```

Comment:
These are assignments. Where do you get this "pre-initialized" stuff from? Did you read my explanation of `const` in the other topic?

http://marlinfw.org/docs/development/coding_standards.html